corehq/apps/locations/resources/v0_6.py

```diff
@@ -57,14 +57,14 @@ def build_filters(self, filters=None, **kwargs):
         if not filters:
             return {}
 
+        # Turn last_modified.gte to last_modified__gte
+        filters = {'__'.join(k.split('.')): v for k, v in filters.items()}
```

Comment:
I'm confused why we moved this earlier? `pop` will remove items that would never be processed by the loop here, so it seems weird to move it earlier and iterate over more items.